Danggn/Screen/Home/Controller/ListItemViewController.swift

```diff
@@ -8,10 +8,105 @@
 import UIKit
 
 class ListItemViewController: UIViewController {
-
+    
+    @IBOutlet weak var listCollectionView: UICollectionView!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        setNavigationUI()
+        registerCell()
+    }
+    
+    @IBAction func itemCreateButtondidTap(_ sender: Any) {
+        
+    }
+}
+
+extension ListItemViewController {
+    func setNavigationUI() {
+        
+        let navigationBar = UINavigationBarAppearance()
+        navigationBar.shadowColor = .gray
+        self.navigationController?.navigationBar.standardAppearance = navigationBar
+
+        let search = navigationItem.setRightButtonUI(self, imageName: UIImage(named: "icon_search"))
+        let menu = navigationItem.setRightButtonUI(self, imageName: UIImage(named: "icon_menu"))
+        let bell = navigationItem.setRightButtonUI(self, imageName: UIImage(named: "icon_bell"))
+    
+        let spacer = UIBarButtonItem(barButtonSystemItem: .fixedSpace, target: nil, action: nil)
+        spacer.width = 16
+        navigationItem.rightBarButtonItems = [bell, spacer, menu, spacer, search]
+    
+        // LeftBarButtonItem 설정
+        // 여백을 주기 위해 동네명을 UIButton으로 만들고 UIView에 담아 myArea라는 UIBarButtonItem애 customView로 대입
+        let myAreaButton = UIButton(frame: CGRect(x: 10, y: 0, width: 50, height: 50))
+        myAreaButton.setTitle("영통동", for: .normal)
+        myAreaButton.setTitleColor(.label, for: .normal)
+        myAreaButton.sizeToFit()
+        
+        let leftBarView = UIView(frame: CGRect(origin: .zero, size: CGSize(width: 100, height: 50)))
+        leftBarView.addSubview(myAreaButton)
+        
+        let myArea = UIBarButtonItem(customView: leftBarView)
+        navigationItem.leftBarButtonItem = myArea
+    }
+}
+
+extension ListItemViewController {
+    
+    // CollectionViewCell 등록
+    func registerCell() {
+        let nib = UINib(nibName: "ListItemCollectionViewCell", bundle: nil)
+        listCollectionView.register(nib, forCellWithReuseIdentifier: "ListItemCollectionViewCell")
```

Comment:
[이곳](https://github.com/THE-SOPT-Joint-Seminar-Danggn/Daangn-iOS/pull/7/files#diff-b515d9f2a420da93bf741b6c5d9f8ec8b3f9645a2783e3f6653c01a1e6f51c86R12)에 담인님께서 만들어놓으신 identifier를 사용하시면 문자열 오타 혹은 누락으로 인한 Runtime Error를 피할 수 있을 것 같네요!!

```suggestion
        let nib = UINib(nibName: ListItemCollectionViewCell.identifier, bundle: nil)
        listCollectionView.register(nib, forCellWithReuseIdentifier: ListItemCollectionViewCell.identifier)
```

Reply:
프로오타러 뜨끔 하고 갑니다

Reply:
혹시 이곳이 어디를 가르키는 것인가요..?ㅎㅎ

Reply:
알려주신대로 identifier를 이용해 코드수정하였습니다! 이게 오타를 줄이고 훨씬 좋은것 같네요!

Reply:
@daminoworld 
> 혹시 이곳이 어디를 가르키는 것인가요..?ㅎㅎ
line number를 link걸어놓았습니다.
바로 화면에 line이 highlighted되지 않는다면 3~4초 기다리시는 걸 권장드립니다:)

Reply:
@noah0316 그런데 세미나 자료에서는 identifier를 셀을 구분하는 id라고 생각하면 된다고 적혀 있는데 identifire를 사용하는 이유가 셀을 구분하는 역할을 함과 동시에 문자열 오타 혹은 누락의 역할도 하는 건가요??